### PR TITLE
Add Azure ML running to the face segmentation task.

### PR DESCRIPTION
**Pull request by lovettchris**
# Pull Request Template

## Description

This adds a script that runs the entire face segmentation search pipeline in Azure ML using both remote SNPE device testing and remote partial training on Azure ML.

Fixes: # (issue)

## Changes

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to not work as expected)
- [ ] Documentation update

**Configuration**:
* Archai version:
* Environment:
* OS:

## Final checks:

- [ ] Follows guidelines of project
- [ ] Self-review of code
- [ ] Commented code in hard-to-understand areas
- [ ] Corresponding changes to the documentation
- [ ] Changes generate no new warnings
- [ ] Dependent changes have already been merged
- [ ] Checked code correction any misspellings

### CODE REVIEW

File: CODEOWNERS
Comment (gugarosa) [inline CODEOWNERS:1]: I think it would be great if you are also able to be the codeowner @lovettchris. What do you think?

**Review by gugarosa — APPROVED** [review]

File: CODEOWNERS
Comment (lovettchris) [reply CODEOWNERS:1]: happy to own any code I wrote in this AML and SNPE section of the repo, how do we do that?

Comment (gugarosa) [reply CODEOWNERS:1]: We can specify a subsection with folders from the `face_segmentation`:

```
## Face Segmentation
/tasks/face_segmentation/aml @lovettchris 
...
```

What do you think?

Comment (lovettchris) [reply CODEOWNERS:1]: done.

**Pull request merged**